src/WebUI/Common/KubeSummary.scss

```diff
@@ -1,15 +1,37 @@
 .kubernetes-container {
     // cards have margins
-    .k8s-card-padding+.k8s-card-padding {
+    .k8s-card-padding + .k8s-card-padding {
         margin-top: 16px;
     }
-
     /*Css to adjust the alignment of text between picklist and text search. Revisit when we move from picklist to dropdown (fabric-free)*/
-    .keyword-search.bolt-text-field{
+    .keyword-search.bolt-text-field {
         padding-bottom: 8px;
     }
 
     .loading-pods {
         height: 200px;
     }
-}
\ No newline at end of file
+
+    .details-card-content {
+        flex-wrap: wrap;
+        width: 100%;
+
+        .details-card-row-size {
+            padding-top: 6px;
+            padding-bottom: 6px;
+            display: flex;
+            position: relative;
+        }
+
+        .details-card-info-field-size {
+            width: 96px;
+            min-width: 96px;
+        }
+
+        .details-card-value-field-size {
+            position: absolute;
+            width: calc(100% - 136px);
```

Comment:
add a comment to say, label has left of 20px and we want to have 20px left of value, so total 40px + width of label. that is why 136px

Reply:
Added comment for explaining 136px logic